.github/workflows/stale.yml

```diff
@@ -0,0 +1,17 @@
+name: "Handle stale PRs"
+on:
+  schedule:
+    - cron: "30 7 * * 1-5"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          only: pulls
+          stale-pr-message: "This PR hasn't seen activity in two months! Should it be merged, closed, or worked on further? If you want to keep it open, post a comment or remove the `stale` label – otherwise this will be closed in a fortnight."
+          close-pr-message: "This PR was closed due to 2 months of inactivity. Feel free to reopen it if still relevant."
+          days-before-pr-stale: 60
+          days-before-pr-close: 14
```

Comment:
Thought here was 2 months of no activity -> 14 days to do something about it. I would do a week but the way our pairing works, this may be a less strict way.

Reply:
Feels about right to me. Do you know if this will retroactively apply `stale` labels to all PRs that meet this criterion?

Reply:
That is a good question! I can't seem to find any documentation around that. From what I understand it should though but wouldn't be entirely surprised If it doesn't - potentially caches all of them and keeps a tally from when it first runs?

If it doesn't I'm happy to do that manually as it's only ~50.